awx_collection/tools/vars/associations.yml

```diff
@@ -0,0 +1,16 @@
+---
+associations:
+  job_templates:
+    - related_item: credentials
+      endpoint: credentials
+      description: "The credentials used by this job template"
+      required: false
+  groups:
+    - related_item: hosts
+      endpoint: hosts
+      description: "The hosts associated with this group"
+      required: false
```

Comment:
I think `required` shouldn't be a thing here, and the arg spec should just leave it out.